source/User_Guide/Settings/Whitelabel/How_to_set_up_link_branding.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+weight: 0
+title: How to set up link branding
+seo:
+  title: How to set up link branding
+  description: Email link branding allows all of the click-tracked links and opens tracked images in your emails to be from your domain instead of from sendgrid.net.
```

Comment:
is it opens-tracked?

Reply:
double checked this and I am using it correctly - open track image is that tiny jpg they use for open tracking